internal/local/structure_v1.go

```diff
@@ -0,0 +1,276 @@
+package local
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+)
+
+// AppStructureV1 represents the v1 Realm app structure
+type AppStructureV1 struct {
+	ConfigVersion        realm.AppConfigVersion   `json:"config_version"`
+	ID                   string                   `json:"app_id,omitempty"`
+	Name                 string                   `json:"name"`
+	Location             realm.Location           `json:"location"`
+	DeploymentModel      realm.DeploymentModel    `json:"deployment_model"`
+	Security             map[string]interface{}   `json:"security"`
+	Hosting              map[string]interface{}   `json:"hosting,omitempty"`
+	CustomUserDataConfig map[string]interface{}   `json:"custom_user_data_config"`
+	Sync                 map[string]interface{}   `json:"sync"`
+	Environment          interface{}              `json:"environment,omitempty"` // TODO realm.Environment
```

Comment:
should this TODO have a ticket attached?

Reply:
nope, i had this as a reminder for me to go and figure out what this was in the backend.  we can use just `string` for now